src/sdk/conference/client.js

```diff
@@ -350,6 +350,9 @@ export const ConferenceClient = function(config, signalingImpl) {
   // eslint-disable-next-line require-jsdoc
   function clean() {
     participants.clear();
+    for (const [id, stream] of remoteStreams.entries()) {
+      stream._stop();
```

Comment:
I don't see the rationality of firing Ended event on a RemoteStream that's not subscribed.  Please reconsider how to define the onEnded event.

Reply:
If you want to know when a subscribed `MediaStream` is ended, please add listener to `RemoteStream.stream`. It makes sense to keep the SDK's behavior as is since the stream is still there in the conference. I'll update the doc to make it clear when `ended` event is fired.